delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/mu/bio/BioSuite.scala

```diff
@@ -14,6 +15,7 @@ abstract class BioSuite
     with BioAssertions
     with BIOValues
     with BIOStreamAssertions
+    with CatsIOValues
```

Comment:
Is this definitely good? I guess I can't see a problem that would occur

Reply:
It is also temporary, the thing is acl check now returns a CatsIO and is used by some module that are not yet migrated